lib/blocks.js

```diff
@@ -0,0 +1,50 @@
+const { Vec3 } = require('vec3')
+
+const CARDINAL_OFFSETS = {
+  north: new Vec3(0, 0, -1),
+  south: new Vec3(0, 0, 1),
+  west: new Vec3(-1, 0, 0),
+  east: new Vec3(1, 0, 0)
+}
+
+const CHEST_FACING_MAP = {
+  north: { west: 'right', east: 'left' },
+  south: { west: 'left', east: 'right' },
+  west: { north: 'left', south: 'right' },
+  east: { north: 'right', south: 'left' }
+}
+
+function parseChestMetadata (bot, chestBlock) {
```

Comment:
Why the move of these functions?

Reply:
Nvm I didn't noticed they had to access bot properties and decided to move them into a library, but then I noticed and forgot to move them back